src/StackAdmin/Azs.Backup.Admin/Help/Restore-AzsBackup.md

```diff
@@ -97,6 +97,36 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -DecryptionCertPath
+Path to the decryption cert file with public key (.pfx).
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases: 
+
+Required: True
```

Comment:
If this is required, then it is a breaking change to the existing cmdlet.. Are we fine with that?

Reply:
As discussed, we'll take these breaking changes and document these well in documentations.